draft-schwartz-ds-glue.md

```diff
@@ -192,39 +198,38 @@ example    3600 IN NS    ns1.example
 ns1.example 600 IN AAAA  2001:db8::1
 ~~~
 
-A zone in this configuration can optionally use an NSEC DSGLUE record to indicate that there is no IPv4 address:
+A zone in this configuration can optionally use an empty DSGLUE record to indicate that there is no IPv4 address:
 
 ~~~
 $ORIGIN com.
-example 600 IN DS $DSGLUE(., NS, ns1.example.com.)
-            IN DS $DSGLUE(ns1., AAAA, 2001:db8::1)
-            IN DS $DSGLUE(ns1., NSEC, ns1.example.com. AAAA)
+example 600 IN DS $DSGLUE(., NS, 3600, [ns1.example.com.])
+            IN DS $DSGLUE(ns1., AAAA, 600, [2001:db8::1])
+            IN DS $DSGLUE(ns1., A, 7200, [])
 ~~~
 
-This arrangement prevents an adversary from inserting forged records for ns1.example.com or _dns.ns1.example.com into the delegation response.
+This arrangement prevents an adversary from inserting forged A records for ns1.example.com into the delegation response.
```

Comment:
How does this matter? If the resolver is processing the DSGLUE records, will it not automatically ignore the unprotected A/AAAA records in the ADDL section in the response?